### DISCUSSION TITLE
Problems encountered by a Nuxt newbie！！

### DISCUSSION THREAD

**Discussion by 1226085293** [nuxt/nuxt#26416] [Questions]
1. scss is recognized as css in the vue file, so I can only create a new scss file.
![image](https://github.com/nuxt/nuxt/assets/45589707/691d88ea-db8b-4e6f-abe2-8bff71ffc349)

2. The global css configuration is invalid and can only be configured using vite.csspreprocessorOptions.scss.additionalData https://github.com/nuxt/nuxt/issues/13690

---

```package.json
"dependencies": {
     "nuxt": "^3.11.1",
     "vue": "^3.4.21",
     "vue-router": "^4.3.0"
   },
   "devDependencies": {
     "node-sass": "^9.0.0",
     "sass": "^1.72.0",
     "sass-loader": "^14.1.1"
   }
```

vscode plugins: Nuxtr, Sass (.sass only), Vetur

**Comment by 1226085293** [comment]
The first problem has been solved. You need to set "files.associations" in .vscode/settings.json to { "*.vue": "vue" }

![image](https://github.com/nuxt/nuxt/assets/45589707/101bca1d-2187-4a6d-942f-287d6806287e)

**Comment by AloisSeckar** [comment]
Vetur is an outdated extension for Vue 2, you should use https://marketplace.visualstudio.com/items?itemName=Vue.volar now
